docs/docs/1.1.1/networking/transparent-proxying.md

```diff
@@ -53,6 +53,10 @@ The default settings will exclude the SSH port `22` from the redirection, thus a
 
 The changes will persist over restarts, so this command is needed only once. Reverting back to the original state of the host can be done by issuing `kumactl uninstall transparent-proxy`.
 
+#### firewalld support
+
+On hosts where `firewalld` is used as a firewall manament and `iptables` wrapper, `kumactl install transparent-proxy` provides a dedicated flag `--store-firewalld` which will make the relevant rules persist across host restarts. The changes are being stored in `/etc/firewalld/direct.xml`, and there is no `uninstall` counterpart of the installation procedure.
```

Comment:
Let's take this up a heading level (H4 does not add value), and copyedit a bit. Guessing a little about "manament" which ... isn't a word in English ;). I might have mangled meaning a little though; lmk if I should tweak further.

#### firewalld support

If you run `firewalld` to manage firewalls and wrap `iptables`, add the `--store-firewalld` flag to `kumactl install transparent-proxy`. This persists the relevant rules across host restarts. The changes are stored in  `/etc/firewalld/direct.xml`. There is no uninstall command for this feature.